tools/utils/http-helpers.js

```diff
@@ -14,7 +14,7 @@ var release = require('../packaging/release.js');
 var Console = require('../console/console.js').Console;
 var timeoutScaleFactor = require('./utils.js').timeoutScaleFactor;
 
-import { WritableStreamBuffer } from 'stream-buffers';
+import concatStream from 'concat-stream';
```

Comment:
The default export is the `ConcatStream` constructor, so let's capitalize the first `C` and use `new` below.